assets/translations/de.json

```diff
@@ -0,0 +1,343 @@
+
+    "@non_native_token":{
+    },
+    "non_native_token_token": "Asset {}",
+    "no_asset_history": "Keine Extrinsics gefunden",
+    "mint_asset_history_label": "Mint {} {}",
+    "mint_asset_history_prefix": "Für:",
+    "set_metadata_asset_history_label": "Metadaten für assetID festlegen: {}",
+    "set_metadata_asset_history_name": "Name: {}",
+    "set_metadata_asset_history_symbol": "Symbol: {}",
+    "set_metadata_asset_history_decimals": "Dezimalen: {}",
+    "non_native_history_title_label": "Geschichte",
+    "create_token_widget": "Token erstellen",
+    "nnt_transfer_button_label": "Assets übertragen",
+
+    "@recieve_page":{
+    },
+    "recieve_page_title": "Empfangen",
+
+    "@transfer_page":{
+    },
+    "transfer_page_title": "Transaktion",
+    "from_address_label": "Von",
+    "to_address_label": "An Adresse",
+    "to_address_hint": "Adresse eingeben",
+    "amount_label": "Menge (Kontostand: {})",
+    "amount_hint": "Menge eingeben",
+    "enter_password_hint": "Passwort eingeben",
+    "enter_password_label": "Passwort",
+    "existential_deposit_label": "existenzielle Einlage",
+    "existential_deposit_text": "Der Mindestbetrag, den ein Account haben sollte, um als aktiv zu gelten.",
+    "estimated_transfer_fee": "Geschätzte Transaktionsgebühr",
+    "keep_alive_label": "am Leben erhalten",
+    "keep_alive_text": "Mit der Option \"Am Leben erhalten\" ist der Account vor der Löschung aufgrund eines niedrigen Kontostands geschützt.",
+    "make_transfer_label": "Transaktion vornehmen",
+    "error_wrong_amount": "Erwartet wurde eine reale Zahl, die kleiner ist als der verfügbare Kontostand.",
+    "could_not_check_address": "Adresse konnte nicht überprüft werden",
+    "wrong_address": "Ungültige Adresse angegeben",
+    "transfer_loader_text": "Übertragen...",
+    "transfer_success_text": "Mittel werden gesendet",
+
+    "error_empty": "Sollte nicht leer sein",
+
+    "@transactions_history":{
+    },
+    "transactions_to": "An",
+    "transactions_from": "Von",
+    "transactions_title": "Transaktionshistorie",
+    "show_transfers": "Transaktionen anzeigen",
+    "no_transfers": "Keine Transaktionen gefunden",
+    "first_page_error_title": "Etwas ist schief gelaufen",
+    "first_page_error_message": "Die Anwendung ist auf einen unbekannten Fehler gestoßen.\\nBitte versuchen Sie es später noch einmal.",
+    "first_page_error_button": "Erneut versuchen",
+
+    "@Screen_lock":{
+    },
+    "screen_lock_title": "PIN-Code eingeben",
+    "screen_lock_title_set": "Legen Sie Ihren PIN-Code fest",
+    "screen_lock_title_confirm": "Bestätigen Sie Ihren PIN-Code",
+
+    "@2.7.2":{
+    },
+    "choose_transfer_keep_alive": "Absender am Leben erhalten",
+    "transfer_keep_alive_help": "Bei aktivierter \"Am Leben erhalten\"-Option ist der Account vor einer Löschung aufgrund eines niedrigen Kontostands geschützt.",
+
+    "@2.8.0":{
+    },
+    "notifications_titile": "Transaktionen",
```

Comment:
Oops. This was a typo in my `en.json`. I fixed it. Please, change to `notifications_title`